JASP-Engine/JASP/R/commonerrorcheck.R

```diff
@@ -335,13 +335,16 @@
     result <- func(dataset[[.v(target)]])
     
   } else {
-    
-    result <- plyr::ddply(dataset, .v(grouping), 
-      function(data, target) {
-        if (any(is.na(data[.v(grouping)])) == FALSE) {
-          func(data[[.v(target)]])
+
```

Comment:
Not really a part of this PR, but can we use the same structure using plyr in the above clause.